interoperability-tests/intel-sgxsdk-cert.pem

```diff
@@ -0,0 +1,207 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
FYI: I looked at this example certificate via `openssl x509 -in intel-sgxsdk-cert.pem -text`. Looks good.

Reply:
thanks a lot for the review, I have updated the document according to your comments.